__tests__/app.test.js

```diff
@@ -95,4 +95,44 @@ describe('GET /api/articles/', () => {
             });
         });
     });
+
+    describe('/api/articles', () => {
+        test('GET 200: should respond with an articles array of article objects, each of which should have the following properties: author, title, article_id, topic, created_at, votes, article_img_url, comment_count, (total count of all the comments with this article_id)', () => {
+            return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({body}) => {
+                expect(body.length).toBe(12);
+                body.forEach((article) => {
+                    expect(Object.keys(article)).toEqual(['article_id', 'title', 'topic', 'author', 'created_at', 'votes', 'article_img_url', 'comment_count']);
```

Comment:
This could be a little better, you're looping through all the objects and then looping the objects again to get the keys. You can assert the object matches an object containing those keys and data types.